src/Microsoft.AspNetCore.Mvc.TagHelpers/Internal/CryptographyAlgorithms.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Security.Cryptography;
+
+namespace Microsoft.AspNetCore.Mvc.TagHelpers.Internal
```

Comment:
The only issue with that is we'd have to account for the TFMs the library is targeting. i.e. if your project is targeting net451 and netstandard1.3, we'd need to ifdef for those two. Failing which on net46, net461 which have the bug we'd use the netstandard code path.